parsel/csstranslator.py

```diff
@@ -89,6 +101,116 @@ def xpath_text_simple_pseudo_element(self, xpath):
         """Support selecting text nodes using ::text pseudo-element"""
         return XPathExpr.from_xpath(xpath, textnode=True)
 
+    def xpath_first_of_pseudo(self, xpath):
```

Comment:
ahh, got it now! thanks for explaining! :)
yea, I suppose `nth-of` is a good name after all! :+1: 

so, why is it WIP? is there something missing for it to be merged?
any idea why it failed it with pypy in Travis?